proc-macros/src/lib.rs

```diff
@@ -225,27 +225,27 @@ fn build_client_functions(api: &api_def::ApiDefinition) -> Result<Vec<proc_macro
 			params_to_json.push(quote_spanned!(pat_span=>
 				map.insert(
 					#rpc_param_name.to_string(),
-					jsonrpsee_types::jsonrpc::to_value(#generated_param_name.into()).map_err(|e| jsonrpsee_types::error::Error::Custom(format!("{:?}", e)))?
+					jsonrpsee::types::jsonrpc::to_value(#generated_param_name.into()).map_err(|e| jsonrpsee::types::error::Error::Custom(format!("{:?}", e)))?
```

Comment:
note, this will only work when using it from `jsonrpsee` root crate which is the most likely use-case.

worth fixing this for when using the internal crate `jsonrpsee_proc_macros` individually?!

Reply:
I think it's fine and even good to make the `jsonrpsee::types` crate only usable with the root crate. We don't really want to stick to semver for the `types` crate anyway; it's really more of an internal-use crate.

Reply:
Fixed that by using [Basti's proc macro crate](https://docs.rs/proc-macro-crate/1.0.0/proc_macro_crate/) instead